api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/v3/EligibilityAndEntitlementServiceV3.java

```diff
@@ -40,15 +42,15 @@
     public EligibilityAndEntitlementDecision evaluateNewClaimant(Claimant claimant) {
         log.debug("Looking for live claims for the given NINO");
         Optional<UUID> liveClaimsWithNino = claimRepository.findLiveClaimWithNino(claimant.getNino());
+        if (liveClaimsWithNino.isPresent()) {
+            return buildDuplicateDecisionWithExistingClaimId(liveClaimsWithNino.get());
+        }
```

Comment:
I don't understand why this has changed because of removing the `updateClaim` functionality?

Reply:
In the case of a duplicate nino there's no need to perform the eligibility check as we already know it's a duplicate. So the live nino check has been moved up in the method.

Reply:
I think my point was that this should probably have been in its own PR, it sticks out like a sore thumb in this PR.